lib/board.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class Board
+  attr_accessor :board
```

Comment:
We don't seem to be mutating `board` anywhere. So what do you think about just making this `attr_reader` instead? We must take care in making our object attributes mutable by other objects. We must, in fact, minimize the number of public interfaces to those that are absolutely necessary (public interfaces means object public instance variables and public methods). And for those that are absolutely necessary to be public, we must by default make them just readable until a need arises to make them writeable.